docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine AS build
+WORKDIR /pack
+COPY --chown=1000 . .
+RUN npm pack
+
+FROM node:lts-alpine AS prod
+# hadolint ignore=DL3010
+COPY --from=build /pack/markdownlint-cli2-*.tgz /
+RUN npm install --global --no-package-lock --production markdownlint-cli2-*.tgz && \
+    rm /markdownlint-cli2-*.tgz
+
+USER node
+
+ENTRYPOINT ["/usr/local/bin/markdownlint-cli2"]
```

Comment:
So I see that, yes, both `markdownlint-cli2` and `markdownlint-cli2-fix` are available in the resulting container. However, by definition, a container can only have a single `ENTRYPOINT`.

Since this line sets the default `ENTRYPOINT` as `markdownlint-cli2`, a user that would want to call the `markdownlint-cli2-fix` instead would have to execute the container by explicitly specifying it as the entrypoint; like this:

```bash
$ docker run --rm -v $PWD:/app -w /app --entrypoint="markdownlint-cli2-fix" davidanson/markdownlint-cli2 "**/*.md" "#node_modules"
```

As an alternative, we _could_ have the container execute `markdownlint-cli2-fix` instead of `markdownlint-cli2`  if the user passed a  `--fix` command line arg; something like:

```bash
$ docker run --rm -v $PWD:/app -w /app davidanson/markdownlint-cli2:latest --fix "**/*.md" "#node_modules"
```
However, the `--fix` argument would then be passed along to the tool:
```bash
markdownlint-cli2 v0.2.0 (markdownlint v0.24.0)
Finding: --fix **/*.md !node_modules
...
```
Which from my understanding will look for a file/directory called `--fix` to lint.

IMHO, stick with explicitly specifying the `ENTRYPOINT` for now. If you eventually want the container to offer both _as a single tool_, then `markdownlint-cli2` itself should also offer it _as a single tool_. I mean, instead of providing a separate `markdownlint-cli2-fix` executable, incorporate it as an argument that you pass to `mardownlint-cli2` itself. Many other linters do it this way (e.g. prettier.js). This could be a future issue/PR if you deem it appropriate.

What we should do, is update the README.md to include docker usage scenarios, including this one to call `markdownlint-cli2-fix`.